### DISCUSSION TITLE
PR still waiting for review after two days

### DISCUSSION THREAD

**Discussion by lelegard** [microsoft/winget-pkgs#192170] [General]
Hi,

This is probably not the first question of that kind but there is no definite answer : how long a PR is usually waiting for review in average? Responses vary but there is a consensus around an hour or so. This is also my experience.

I have submitted PR #191987 more than two days ago. This is not a new package, just a new version of an existing package. The PR was created using `wingetcreate update`. There is no change in the package manifests, apart from the version. So, this is the most basic PR possible.

The status is "No unresolved conversations", "All checks have passed" and blocked on "Review required", "Merging is blocked".

All previous versions were reviewed and merged within an hour. So, what's going on these days? Any idea?

**Comment by lelegard** [comment]
About 15 minutes after I posted this message, PR #191987 was approved 😲. This must be a only a coincidence because if it was the result of a reviewer reading this and unblocking the PR, he/she would have kindly acknowledged the problem here. Well, I guess...

**Reply by Trenly** [reply]
@lelegard - Approvals generally are slower on the weekends as both the MSFT employees and the volunteer moderators take time with their families and/or to relax. I'm sure that the approval timing in this case was just a coincidence. 

There is no definitive time that a PR spends waiting for review - it really depends on when it was submitted, who is available for reviews, and how complex the reviews are. Sometimes you get lucky and a moderator is reviewing PRs at the same time it's submitted, other times the moderators are all busy and unable to check in. The longest I've seen a PR to review is 3 days (due to a weekend + a holiday), but I would say that 90% of PRs are reviewed within 24 hours.

If you have an application that is critical to get an update out immediately - for example, some community members maintain Google Chrome, where it's completely unavailable until the PR for the new version merges - I would recommend joining the [Gitter chat](https://gitter.im/Microsoft/winget-pkgs) linked in the [README](https://github.com/microsoft/winget-pkgs/blob/master/README.md). Most of the moderators watch that chat, even on the weekends, in case something pops up that they need to approve a PR quickly
